src/operator/api/v1alpha2/intents_types.go

```diff
@@ -217,12 +217,34 @@ func (in *ClientIntentsList) FormatAsOtterizeIntents() ([]*graphqlclient.IntentI
 	for _, clientIntents := range in.Items {
 		for _, intent := range clientIntents.GetCallsList() {
 			input := intent.ConvertToCloudFormat(clientIntents.Namespace, clientIntents.GetServiceName())
+			input.Status = clientIntentsStatusToCloudFormat(clientIntents)
+
 			otterizeIntents = append(otterizeIntents, lo.ToPtr(input))
 		}
 	}
+
 	return otterizeIntents, nil
 }
 
+func clientIntentsStatusToCloudFormat(clientIntents ClientIntents) *graphqlclient.IntentStatusInput {
+	var status graphqlclient.IntentStatusInput
+
+	serviceAccountName, ok := clientIntents.Annotations[OtterizeClientServiceAccountAnnotation]
+	if !ok {
+		return nil
+	}
+
+	status.ServiceAccountName = toPtrOrNil(serviceAccountName)
+	value, ok := clientIntents.Annotations[OtterizeSharedServiceAccountAnnotation]
```

Comment:
You can shorten this block by just doing if ok { lo.ToPtr }, since nil is the default value

Reply:
I like it to be explicit and in the same format of `if !ok {` same as line above